src/Core/Prefab/GlobeView.js

```diff
@@ -247,6 +247,12 @@ function GlobeView(viewerDiv, coordCarto, options = {}) {
 
     const renderer = this.mainLoop.gfxEngine.renderer;
 
+    const coordCam = new Coordinates(this.referenceCrs, 0, 0, 0);
```

Comment:
No need to set the CRS to `referenceCRS` as it is getting rewritten to `EPSG:4978` after.

Reply:
crs is mandatory parameter in constructor